README.md

```diff
@@ -12,7 +12,11 @@ Set the `PROXIES` environment variable to a JSON object mapping proxy routes to
 
 ## Deployment
 
-Unlike the other applications, the **`master`** branch is usually deployed to all three environments, and there is no special workflow for any updates or hotfixes. However, it is best to deploy the proxy app at night because it may temporarily effect our routes and wagtail work during deploy.
+The proxy follows [other projects](https://github.com/fecgov/openFEC#creating-a-release) for the release and hotfix processes. The changes will automatically deploy when PR's are merged, or the release is cut and deployed.
+
+However, it is best to deploy the proxy app at night because it may temporarily effect our routes and wagtail work during deploy. For this reason, builds require manual approval in CircleCI to run.
```

Comment:
I'm not sure this is still true, but for now it's better to leave it a bit manual and we can remove that later after more testing.

Reply:
We may not need to worry about nightly deploys anymore, since rolling deployments doesn't delete any apps like in the zero downtime autopilot method, there's just a rolling of a new instance rather than an app. So it shouldn't effect any route bindings that Wagtail uses.

Reply:
👍 thanks @patphongs! Modified with 8e4953a5867d715e24153777babf8bf6c65767ff